packages/components/src/SubHeaderBar/SubHeaderBar.scss

```diff
@@ -0,0 +1,61 @@
+$subheader-bar-height: 7rem;
```

Comment:
component var should be prefixed by `tc-` and have `!default` directive in order to be overrided